definitions/ext-ibmmqqueue/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: EXT
+type: IBMMQQUEUE
+
+synthesis:
+  name: qName
+  identifier: qName
+  conditions:
+    - attribute: eventType
+      value: "MQQueueSample"
+
+tags:
```

Comment:
The tags section needs to be under synthesis

Reply:
@IreneMLC complete!